contributing.md

```diff
@@ -0,0 +1,99 @@
+# Contributing Guidelines
+
+This document will guide you through the process of contributing to our projects and help maintain a healthy and
+collaborative environment.
+
+## Table of Contents
+
+1. [Getting Started](#getting-started)
+2. [Issues](#issues)
+3. [Making Changes](#making-changes)
+    - [Coding Conventions](#coding-conventions)
+4. [Pull Requests](#pull-requests)
+    - [For the Author](#for-the-author)
+    - [For the Reviewer](#for-the-reviewer)
+5. [Testing](#testing)
+6. [Questions](#questions)
+7. [Code of Conduct](#code-of-conduct)
+
+## Getting Started
+
+To get started, please refer to the [bln-dev-handbook](https://github.com/bluedrop-learning-networks/bln-dev-handbook)
+and [bln-ops-handbook](https://github.com/bluedrop-learning-networks/bln-ops-handbook) for an overview of our
+development and operational processes. We highly recommend starting with the [Development Environment - From Zero to
+Tilt](https://github.com/bluedrop-learning-networks/bln-ops-plan-state/blob/master/local/README.md) article, which will
+help you set up an end-to-end local environment for our projects.
+
+Before contributing, make sure to familiarize yourself with the project's documentation, codebase, and any relevant
+issues.
+
+## Issues
+
+We use Jira for issue management. You can find our Jira board at `bluedrop360.atlassian.net`. This is where your tasks
+will be assigned and where work should be planned out. If you encounter a bug, please create a new issue on Jira,
+providing a clear description and any relevant details. Make sure to check for existing issues to avoid duplicates.
+
+## Making Changes
+
+When contributing, please adhere to the typical Git workflow: create branches, submit pull requests, follow coding
+conventions, and ensure your code is appropriately tested. Make sure to have your pull request reviewed and follow the
+checklist provided in the pull request template.
+
+### Coding Conventions
+
+Please follow the eslint rules and .editorconfig rules for formatting and spacing within a project. Consistent coding
+style is essential for readability and maintainability. If you're unsure about a specific coding convention, consult the
+project's documentation or ask for guidance from the team.
+
+## Pull Requests
+
+- Code review is a responsibility shared by all team members, regardless of seniority.
+- Authors are responsible for their own code and ultimately for merging their own code.
+- Generally, at least one approving review is required to merge code. Pull requests containing user-facing functional
+  changes require an approving QC review.
+- Dismiss stale reviews is disabled to allow addressing optional, non-blocking comments on a PR without incurring
+  another round of review. Please refrain from introducing substantive or unrelated changes to a branch after a PR has
+  been approved.
```

Comment:
Do we want to add the instruction on how to keep our repo cleans (delete branches, ...)?

Reply:
a0b4319a1d67220520fe10863914265ad254b771